Modelica/Electrical/Analog/Examples/package.order

```diff
@@ -31,6 +31,6 @@ ParallelResonance
 DemonstrateLightning
 DemoPowerSupply
 DemoPowerSupplyWithBuffer
-Lines
 OpAmps
+Lines
```

Comment:
Also question about the order.

Reply:
Yes I tried to order the examples according to the problems they did rise during the last regression tests ;-)
I see no big harm in changing the order ...